pkg/utils/kube/scheme/scheme.go

```diff
@@ -1,29 +1,106 @@
 package scheme
 
 import (
-	"k8s.io/kubernetes/pkg/api/legacyscheme"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Would it be possible to instead use the Scheme found in `k8s.io/client-go/kubernetes/scheme` (there's probably a good reason it cannot, but would like to understand why)?

Reply:
I think that would actually be preferred. Thanks for pointing to it. It is equivalent.

Reply:
done